contracts/state_rent/IStateRent.sol

```diff
@@ -5,6 +5,12 @@ pragma solidity ^0.8.0;
 interface IStateRent {
     event TransferredMaintainer(address indexed contract_address, address indexed new_maintainer);
 
+    event ContractDeployed(address indexed contract_address);
+
+    event AccountEnabled(address indexed account_address);
+
+    event AccountDisabled(address indexed account_address);
```

Comment:
Maybe `developmentEnabled` is better. Or others.

Reply:
yeah I like that more, AccountDisabled could scare someone haha